src/tsdl_new.c

```diff
@@ -408,3 +408,9 @@ CAMLprim value TSDL_GetWindowMaximumSize(value window) {
 
   CAMLreturn(ret);
 }
+
+CAMLprim value TSDL_ShowCursor(value toggle) {
+  CAMLparam1(toggle);
+  int visible = SDL_ShowCursor(toggle);
```

Comment:
You need to call the macro Int_val on toggle before it’s actually an int, otherwise from C’s perspective it’s wrong.